content/en/docs/tutorials/stateful-application/cassandra.md

```diff
@@ -95,65 +102,80 @@ The StatefulSet manifest, included below, creates a Cassandra ring that consists
 1. Update the StatefulSet if necessary.
 2. Create the Cassandra StatefulSet from the `cassandra-statefulset.yaml` file:
 
-       kubectl create -f cassandra-statefulset.yaml
+    ```bash
+    kubectl create -f cassandra-statefulset.yaml
+    ```
 
 {{< code file="cassandra/cassandra-statefulset.yaml" >}}
 
 ## Validating The Cassandra StatefulSet
 
 1. Get the Cassandra StatefulSet:
 
-       kubectl get statefulset cassandra
+    ```bash
+    kubectl get statefulset cassandra
+    ```
 
-   The response should be
+    The response should be:
 
-       NAME        DESIRED   CURRENT   AGE
-       cassandra   3         0         13s
+    ```
+    NAME        DESIRED   CURRENT   AGE
+    cassandra   3         0         13s
+    ```
 
    The StatefulSet resource deploys Pods sequentially.  
 
 2. Get the Pods to see the ordered creation status:
 
-       kubectl get pods -l="app=cassandra"
+    ```bash
```

Comment:
All of this interior content needs to be indented one tab-stop further so that you don't break the numbered list after `2`. THis is a known Black Friday bug. We always need to check the previews (https://deploy-preview-9196--kubernetes-io-master-staging.netlify.com/docs/tutorials/stateful-application/cassandra/) for things like this, as Black Friday is incredibly brittle. Please check the preview for the rest of the file to make sure no other lists are broken in this way.

Reply:
I definitely always check the preview but I think I was looking for indentation issues and overlooked the numbering problem. As far as I can tell everything is now in order.